submissions/semenjob/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav__list">
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Mail</a>
+        </li>
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Images</a>
+        </li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
+        type="checkbox"
+        id="popup__checkbox"
+      />
+      <label for="popup__checkbox" class="popup__checkbox__label">
+        <img
+          class="popup__icon__head"
+          src="./icon/popup-button.png"
+          alt="button logo"
+        />
+      </label>
+      <nav class="popup">
+        <ul class="popup__list">
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/apps.png" alt="Apps logo" />
+            <span class="icon_text">Apps</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/bell.png" alt="Bell logo" />
+            <span class="icon_text">Bell</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/blogs.png" alt="Blogs logo" />
+            <span class="icon_text">Blogs</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img
```

Comment:
Semantically and logically all these items are links, so need to be wrapped in corresponding tag

Reply:
ok corrected